src/Styles.js

```diff
@@ -16,8 +16,9 @@ export default css`
   display: flex;
   flex-direction: row;
   align-items: center;
-  border-bottom: 1px var(--api-body-document-title-border-color, var(--api-parameters-document-title-border-color, #e5e5e5)) solid;
-  border: var(--api-body-document-title-border);
+  border-bottom: var(--api-body-document-title-border-bottom,
+    1px var(--api-body-document-title-border-color, var(--api-parameters-document-title-border-color, #e5e5e5)) solid
+  );
```

Comment:
What's the initial idea for border styling actually? To make it possible to modify the size of `border-bottom`, or being able to also modify the color of all border sides? If it's about sides, it can be done in the following way:
```css
border: var(--api-body-document-title-border, 1px solid);
border-color: var(--api-body-document-title-border-color-full, transparent var(--api-parameters-document-title-border-color, #e5e5e5) transparent transparent);
```

This way the default behavior is preserved while customization of border becomes more flexible:
1. The border is set to `1px solid` if variable is not defined.
2. The border color is set to `transparent` for all border sides except bottom by default.

Notice that I named variable as `--api-body-document-title-border-color-full`, because there's a variable name clash. I am not sure how it should be handled in this case but maybe other contributors know better.

Reply:
For my use case, I am only styling the border-bottom (actually, hiding it by setting `border-bottom: 0`) so I may not need to color the other border sides

Reply:
@deiteris @benri I am working on a consolidation of all documentation components under a single repository. You can see the new code base here: https://github.com/advanced-rest-client/api-documentation/tree/release/7.0.0
It is not yet ready as I am still merging test cases from the old components. This will be a breaking change anyway so it will be easier to make major styling changes out there. I'll be happy to hear your feedback.

Reply:
@jarrodek since you're moving some stuff from `api-type-document` there as well, please also take a look at the issues and PRs here:
https://github.com/advanced-rest-client/api-type-document/issues